packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -63,10 +64,12 @@ const NavigationBar = ({
       data-test={dataTest}
       id={id}
       className={cx(
-        "bg-white-normal shadow-fixed p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "bg-white-normal p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
         "duration-normal transform-gpu transition-transform ease-in-out",
         "tb:h-1600 h-[52px]", // As defined on the const above
-        shown ? "translate-y-0" : "tb:-translate-y-1600 translate-y-[-52px]", // As defined on the const above
+        shown ? "translate-y-0" : "tb:-translate-y-1600 -translate-y-xxxl", // As defined on the const above
```

Comment:
Out of the scope, but what const?

Reply:
Actually, I don't know why this change happened 🤔 As is said above, `xxxl` is deprecated. But I don't recall making this change 🤯